### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by joelabair** [tojocky/node-printer#92]
```javascript
node
> printer = require('printer');
> printer.printFile({filename: '/tmp/test.txt', type: 'TEXT'})
Segmentation fault
```

On Fedora 20
cups
cups-libs
cups-devel 

bash >node -v
v0.10.36

**Comment by tojocky** [comment]
What node-printer version?

**Comment by joelabair** [comment]
Installed from git.  Says version 0.2.0 in package.json

**Comment by tojocky** [comment]
Resolved in v0.2.1

**Comment by joelabair** [comment]
Thanks!  Just in time.
